geomesa-hbase/geomesa-hbase-datastore/src/main/scala/org/locationtech/geomesa/hbase/index/HBaseFeatureIndex.scala

```diff
@@ -176,8 +176,10 @@ trait HBaseFeatureIndex extends HBaseFeatureIndexType
     import org.locationtech.geomesa.index.conf.QueryHints.RichHints
 
     /** This function is used to implement custom client filters for HBase **/
-      val toFeatures = resultsToFeatures(sft, ecql, hints.getTransform)
-      ScanConfig(Nil, toFeatures)
-
+      val toFeatures = resultsToFeatures(sft, None, hints.getTransform)
+      val remoteFilters = filter.filter.map { filter =>
```

Comment:
@MutahirKazmi yes good point. @jnh5y thoughts? My understanding was we wanted to merge this one first, then merge the PR that toggles the remote filter. That will put us in a temporary head state where filters will always be run.